packages/explat-client/src/types.ts

```diff
@@ -0,0 +1,39 @@
+// ## The data structures
+
+export interface ExperimentAssignment {
```

Comment:
This is the only data structure we need. We use it everywhere, it is extensible.